components/avatar/avatar.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Avatar</title>
+    <link rel="stylesheet" href="/utils.css">
+    <link rel="stylesheet" href="/components/avatar/avatar.css">
+</head>
+<body>
+    <section class="container">
+        <div class="avatar-container size-sm circle" >
```

Comment:
class name such as avatar-circle and avatar-sq will make better it more readable.